plugins/janus_streaming.c

```diff
@@ -721,6 +721,11 @@ rtspiface = network interface IP address or device name to listen on when receiv
 #include "../utils.h"
 #include "../ip-utils.h"
 
+/* Default settings */
+#define JANUS_DEFAULT_SESSION_TIMEOUT 0 /* Overwrite the RTSP session timeout. If set to zero, the RTSP timeout is derived from a session. */
+#define JANUS_DEFAULT_RECONNECT_DELAY 5 /* Reconnecting delay in seconds. */
+#define JANUS_DEFAULT_CURL_TIMEOUT 10L /* Communication timeout for cURL. */
+#define JANUS_DEFAULT_CURL_CONNECT_TIMEOUT 5L /* Conection timeout for cURL. */
```

Comment:
The prefix should be `JANUS_STREAMING_`, not just `JANUS_`, as they're specific to the streaming plugin. The core may have defines with the same name for its own context, otherwise, which would cause problems.

Besides, the comment on the right should be separated by one or more tabs, not a space (code style).